components/config-mgmt-service/grpcserver/cfg_mgmt.go

```diff
@@ -3,6 +3,7 @@ package grpcserver
 import (
 	"context"
 	"strings"
+	"time"
```

Comment:
This contains the main change for this PR. Instead of just checking if the node exists with the allowed projects, we are now pulling the projects that are tagged on the associated node.